when externalStorage.createDatabase=true, apiserver-deployment.yaml init-container SQL syntax error

### What happened?

❯ 
apiserver中init-container 运行失败！日志如下：
kubectl logs -n clusterpedia-system clusterpedia-apiserver-699d9b997c-rf7wx -c ensure-database
mysql: Can't get stat of '/opt/bitnami/mysql/conf/bitnami/my_custom.cnf' (OS errno 2 - No such file or directory)
mysql: [ERROR] Stopped processing the 'include' directive in file /opt/bitnami/mysql/conf/my.cnf at line 30.
mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '{database}' at line 1
waiting for database check

检查apiserver-deployment.yaml中init-container. command 发现是由于单引号导致${database}没有被转移导致无法引用正确的数据库名称。

### What did you expect to happen?

修改apiserver-deployment.yaml 中 init-container. command内容，修改逻辑可以参考clustersynchro-manager-deployment.yaml文件

### How can we reproduce it (as minimally and precisely as possible)?

 当设置.Values.externalStorage.createDatabase=true时，无法成功运行apiserver。

### Anything else we need to know?

_No response_

--- a/charts/clusterpedia/Chart.yaml
+++ b/charts/clusterpedia/Chart.yaml
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.6.0
+version: 1.6.1
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to

--- a/charts/clusterpedia/templates/apiserver-deployment.yaml
+++ b/charts/clusterpedia/templates/apiserver-deployment.yaml
@@ -41,7 +41,7 @@ spec:
             # Load YAML data into a Bash variable
             source /opt/scripts/parse-yaml.sh;
             create_variables /etc/clusterpedia/storage/internalstorage-config.yaml;
-            until psql -U ${user} -h ${host} -p ${port} postgres -c "SELECT 1 FROM pg_database WHERE datname = ${database}" | grep -q 1 || psql -U ${user} -h ${host} -p ${port} postgres -c "CREATE DATABASE ${database} owner ${user} " -c "GRANT ALL PRIVILEGES ON DATABASE ${database} to ${user}"; do 
+            until psql -U ${user} -h ${host} -p ${port} postgres -c "SELECT 1 FROM pg_database WHERE datname = ${database}" | grep -q 1 || psql -U ${user} -h ${host} -p ${port} postgres -c "CREATE DATABASE ${database} owner ${user} " -c "GRANT ALL PRIVILEGES ON DATABASE ${database} to ${user}"; do
             echo waiting for database check && sleep 1;
             done;
             echo 'DataBase OK ✓'
@@ -51,8 +51,8 @@ spec:
             source /opt/scripts/parse-yaml.sh;
             create_variables /etc/clusterpedia/storage/internalstorage-config.yaml;
             if [ -z $password ]; then password=${DB_PASSWORD}; fi;
-            until mysql -u${user} -p${password} --host=${host} --port=${port} -e 'CREATE DATABASE IF NOT EXISTS ${database}'; do
-            echo waiting for database check && sleep 1; 
+            until mysql -u${user} -p${password} --host=${host} --port=${port} -e "CREATE DATABASE IF NOT EXISTS ${database}"; do
+            echo waiting for database check && sleep 1;
             done;
             echo 'DataBase OK ✓'
           {{- end }}

